src/main/webapp/index.html

```diff
@@ -6,6 +6,7 @@
     <link rel="stylesheet" href="/styles/base.css">
     <link rel="stylesheet" href="/styles/index.css">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Oswald&display=swap" rel="stylesheet">
+    <script src="scripts/cookies.js"></script>
```

Comment:
[Optional] Move the script tag to the end of the body, as it isn't needed during loading.